cmd/zed/Makefile.am

```diff
@@ -22,7 +24,11 @@ zed_SOURCES = \
 	zed_log.c \
 	zed_log.h \
 	zed_strings.c \
-	zed_strings.h
+	zed_strings.h \
+	agents/zfs_agents.h \
+	agents/zfs_diagnosis.c \
+	agents/zfs_mod.c \
+	agents/zfs_retire.c
```

Comment:
You'll want to add the following to the top of this Makefile.am.  Then you can drop the `agents` subdir prefix, this is needed to support VPATH style builds.

```
VPATH = 
        $(top_srcdir)/cmd/zed
        $(top_srcdir)/cmd/zed/agents
```